mastodon/src/main/res/values-de-rDE/strings_mo.xml

```diff
@@ -75,4 +75,6 @@
     <string name="mo_setting_uniform_summary">App-Symbol für alle Benachrichtigungen verwenden</string>
     <string name="mo_setting_reduced_motion_summary">Animationen für Interaktionen deaktivieren</string>
     <string name="mo_setting_play_gif_summary">GIFs in Profilbildern und Emojis automatisch abspielen</string>
+    <string name="mo_settings_show_posts_without_alt">Zeige Medienbeiträge ohne Bildbeschreibung</string>
+    <string name="mo_settings_show_posts_without_alt_summary">Beiräge werden in allen Timelines ausgeblendet, können aber in Threads und Benachrichtungen ausgeklappt werden</string>
```

Comment:
```suggestion
    <string name="mo_settings_show_posts_without_alt_summary">Beiträge werden in allen Timelines ausgeblendet, können aber in Threads und Benachrichtungen angezeigt werden</string>
```
Typo, and I'm not sure about `ausgeklappt`. Maybe use `angezeigt` or `aufgedeckt` instead?